auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.java

```diff
@@ -223,7 +250,7 @@ public AuthenticationRequest login(@NonNull String usernameOrEmail, @NonNull Str
         Map<String, Object> requestParameters = ParameterBuilder.newBuilder()
```

Comment:
I'd keep this as it was, using password grant, since realm grant without a realm is pretty useless. We can recommend using the other method and even deprecating this or explaining the default directory restriction of password grant